util/system/atime_deprecated.go

```diff
@@ -0,0 +1,13 @@
+package system
+
+import (
+	iofs "io/fs"
+	"time"
+
+	"github.com/containerd/continuity/fs"
+)
+
+// Deprecated: use [fs.Atime] instead.
```

Comment:
We don't need this. We don't have any go backwards compatibility guarantees.

Reply:
Removed, although I think "we don't guarantee" and "we force breaking changes with every release" are .. slightly distinct things.